src/main/java/com/box/sdk/BoxAPIConnection.java

```diff
@@ -136,22 +141,32 @@ public class BoxAPIConnection {
   private Authenticator authenticator;
 
   /**
+   * @deprecated
+   * This constructor, and the com.box.sdk package is deprecated, it is recommended to use com.box.sdkgen package.
+   * Instead of this class use {@link com.box.sdkgen.client.BoxClient}
+   *
    * Constructs a new BoxAPIConnection that authenticates with a developer or access token.
    *
    * @param accessToken a developer or access token to use for authenticating with the API.
    */
+  @Deprecated
   public BoxAPIConnection(String accessToken) {
     this(null, null, accessToken, null);
   }
 
   /**
+   * @deprecated
+   * This constructor, and the com.box.sdk package is deprecated, it is recommended to use com.box.sdkgen package.
```

Comment:
Maybe instead of `This constructor` lets put `BoxAPIConnection class and the entire com.box.sdk ...`